tree/ntuple/v7/inc/ROOT/RPageStorage.hxx

```diff
@@ -129,6 +146,11 @@ class RPageSink : public RPageStorage {
 protected:
    RNTupleWriteOptions fOptions;
 
+   /// Helper to zip pages and header/footer; comprises a 16MB unzip buffer.
```

Comment:
```suggestion
   /// Helper to zip pages and header/footer; comprises of a 16MB unzip buffer.
```

Reply:
Is it really "comprise of sth." instead of "comprise sth."?

Reply:
Apparently both are valid with the 'of' version being older and some debate of which is valid or not.  I recommend using a different verb (`consist of` or `includes` or something else to clarify the semantic (i.e. setting asides the 'of' usage, `comprise` is not used that often))